src/app/pages/storage/disks/disk-bulk-edit/disk-bulk-edit.component.html

```diff
@@ -0,0 +1,61 @@
+<ix-modal-header [title]="'Bulk Edit Disks' | translate" [loading]="isLoading"></ix-modal-header>
+<mat-card>
+  <mat-card-content>
+
+    <form [formGroup]="form" class="ix-form-container" (submit)="onSubmit()">
+      <div class="two-columns">
+        <ix-fieldset
+          [title]="helptextBulkEdit.title | translate"
+        >
+          <div class="container-disks">
+            <p class="label-disks">{{helptextBulkEdit.disks.placeholder}}</p>
+            <tooltip *ngIf="helptextBulkEdit.disks.tooltip" [header]="helptextBulkEdit.disks.placeholder" class="tooltip" [message]="helptextBulkEdit.disks.tooltip"></tooltip>
+            <p class="name-disks">
+              <span *ngFor="let name of diskNames;  last as isLast">
```

Comment:
can be simplified into `diskNames.join(', ')`

Reply:
<img width="614" alt="image" src="https://user-images.githubusercontent.com/351613/169756294-039248c0-39cd-41a9-8600-612fa83bd295.png">

The layout looks weird. Maybe we can use disabled `ix-chips` control input for disks and then use one column layout?

Reply:
@undsoft ping

Reply:
Yes to one column layout. We can try chips if they look good and support this, otherwise okay with keeping as text.